Week_01_Basics/homework-portfolio/style.css

```diff
@@ -0,0 +1,100 @@
+nav {
+	text-align: center;
+	color: #707070;
+	font-size: 14pt;
+	font-style: bold;
+	font-family: sans-serif;
+	color: #23635a;
+	padding-left: 50px;
+}
+body {
+	text-align: left;
+	vertical-align: baseline;
+	background: url(./bg.png);
+	max-width: 980px;
+	margin: auto;
+	margin-top: 10px;
+	background-repeat: repeat-x;
+    background-attachment: fixed;
+    background-position: 1% 100%;
+}
+a:link {
+	text-decoration: none;
+}
+a:hover {
+	text-decoration: underline;
+}
+a:visited {
+	color: #707070;
+}
+a {
+	color: #23635a;
+	font-style: bold;
+	font-size: inherit;
+	font-family: sans-serif;
+	color: #23635a;
+}
+h1 {
+	font-size: 25pt;
```

Comment:
pt is used for exact width which is great on print products.  However, on a computer, ideally you want to render based on resolution pixels, so pt is not so great for that.